hms-app/src/components/HackathonSelectDropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect, useState } from 'react'
+import { getListOfHackathons } from '../actions/HackathonActions'
+import { Select } from '@mantine/core'
+import { HackathonPreview } from '../common/types'
+
+type Props = {
+  setHackathonID: (hackthonID: string) => void
+}
+
+export default function HackathonSelectDropdown(setHackathonID: Props) {
```

Comment:
I would name this argument a bit differently.  It is the props you pass in, but not the setHackathonID function itself.  So when you call it later it looks a bit confusing

Reply:
or you can descruct it like `{ setHackathonID }: Props`.  Possibly naming it something like `setHackathonIDCallback` might be good too, just to specify that it is an function from outside this component